store/rootmulti/store.go

```diff
@@ -711,9 +711,9 @@ func (rs *Store) Restore(
 	if height == 0 {
 		return sdkerrors.Wrap(sdkerrors.ErrLogic, "cannot restore snapshot at height 0")
 	}
-	if height > math.MaxInt64 {
+	if height > uint64(math.MaxUint64) {
```

Comment:
Oh oh, now height can be more than math.MaxInt64 aka >9223372036854775808, up until (1<<64 - 1) aka 18446744073709551615. That seems like a breaking change that's not related to ARM compatibility. The error message doesn't even match this check, plus I don't think there was a test to prevent such a regression. Please advise or revert this check.

Reply:
This was changed [here](https://github.com/cosmos/cosmos-sdk/pull/8396#discussion_r561053788) by @alessio. I think he should answer this better

Reply:
This should fix it https://github.com/cosmos/cosmos-sdk/pull/8466 - @odeke-em